.travis.yml

```diff
@@ -1,7 +1,13 @@
 language: python
 python:
   - "3.6"
+services:
+  - docker
 install:
+  - docker build -t max-question-answering-web-app .
+  - docker run -it -d -p 5000:5000 max-question-answering-web-app
   - pip install -r test-requirements.txt
+before_script:
+  - pip install -r requirements.txt
```

Comment:
This should probably go in the `install` section

Reply:
Actually. I don't think this is needed at all. This will be installed inside the docker container, right?